cluster/examples/kubernetes/ceph/kube-registry.yaml

```diff
@@ -19,6 +29,9 @@ spec:
         version: v0
         kubernetes.io/cluster-service: "true"
     spec:
+      # Use the registry's service account, which may or may not have pod security configs
+      serviceAccount: kube-registry
```

Comment:
seems odd that we need both `serviceAccount` and `serviceAccountName` set to the same thing, but I see it other places too. Do you know why we have to set both?

Reply:
I'm not certain that we do, but it was something I saw often in looking online, so I figured it wouldn't hurt.

Reply:
`serviceAccount` is deprecated, you can just use `serviceAccountName`. From https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#podspec-v1-core:

![image](https://user-images.githubusercontent.com/2083930/53742041-66afd680-3ea0-11e9-94f5-25cb71bada97.png)